packages/codecs/src/ipld.ts

```diff
@@ -47,3 +47,27 @@ export const cidAsString = new Type<CID, string, string>(
   },
   (cid) => cid.toString()
 )
+
+/**
+ * codeco codec for CAR file encoded as a Uint8Array.
+ */
+// TODO(2820): Make this codec work
```

Comment:
@ukstv I may need your help to get this working right

Reply:
I filed https://linear.app/3boxlabs/issue/CDB-2520/use-a-codeco-codec-to-deserialize-the-car-file-from-a-uint8array for the a serialization/deserialization coded for CAR files.  Is that something you might be willing to take on?

Reply:
@stbrody https://github.com/ceramicnetwork/codeco/tree/main#idea I have changed README to make it clearer how to create a codec. It would be sad if only Paul and I can do that.

Reply:
I think I understand most of it, the only part I'm not sure of is how to implement the `is` function to tell if a given input is an instance of the CAR class.

Reply:
okay, looked at a few other examples that Paul had done - is it as simple as just using `instanceOf`?  Anyway put up a version that does that